draft-ietf-core-senml.md

```diff
@@ -291,18 +291,19 @@ fields in each Record are the empty string so they are omitted.
 
 Some devices have accurate time while others do not so SenML supports
 absolute and relative times. Time is represented in floating point as
-seconds. Values greater than zero represent an absolute time relative
-to the Unix epoch (1970-01-01T00:00Z in UTC time) and the time is
-counted same way as the Portable Operating System Interface (POSIX)
-"seconds since the epoch" {{TIME_T}}.  Values of 0 or less represent a
-relative time in the past from the current time. A simple sensor with
-no absolute wall clock time might take a measurement every second,
-batch up 60 of them, and then send the batch to a server. It would
-include the relative time each measurement was made compared to the
-time the batch was sent in each SenML Record. The server might have
-accurate NTP time and use the time it received the data, and the
-relative offset, to replace the times in the SenML with absolute times
-before saving the SenML information in a document database.
+seconds. Values greater than 2^28 represent an absolute time relative
```

Comment:
Are we using ^ for exponentiation?  We probably need to point this out somewhere, as C uses ^ for XOR.
(I tend to use ** for exponentiation to keep ^ open for XOR usage.  See end of page 9 of RFC 7252 for a good way to do this.)